### ISSUE TITLE
How I stop zoom on scroll?

### ISSUE THREAD

**Issue by dsushmasagar** [yanwsh/videojs-panorama#140]
I couldn't able to scroll when my cursor is on video. instead of scroll, zoom functionality is working on the video. How do I prevent this?

**Comment by yanwsh** [comment]
does the video host on iframe?

**Comment by dsushmasagar** [comment]
No, It's on video tag. https://plnkr.co/edit/HAFVrEPnw0gt5datTkfV?p=preview
